app/components/school-list.hbs

```diff
@@ -90,27 +90,19 @@
         <table>
           <thead>
             <tr>
-              <th class="text-left" colspan="4">
```

Comment:
Why the `colspan` change?

Reply:
no need to span anymore since it's just one column now.

Reply:
Lol. Yeah. That'll do it!